sorc/build_ww3prepost.sh

```diff
@@ -1,6 +1,29 @@
 #!/bin/sh
 set -x
 
+# Default settings
+APP="S2SWA"
+
+while getopts "a:v" option; do
+  case "${option}" in
+    a) APP="${OPTARG}" ;;
+    v) BUILD_VERBOSE="YES";;
+    *)
+      echo "Unrecognized option: ${1}"
+      exit 1
+      ;;
+  esac
+done
+
+
+# Determine which switch to use 
+if [ APP == ATMW ]; then 
+  ww3switch=model/esmf/switch 
+else 
+  ww3switch=model/bin/switch_meshcap
```

Comment:
Does `$SWITCHFILE` down below need to be updated to use this variable?

Reply:
yes! thank you for catching this too!!